dynamic-plugins/wrappers/backstage-community-plugin-catalog-backend-module-scaffolder-relation-processor/dist-dynamic/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "backstage-community-plugin-catalog-backend-module-scaffolder-relation-processor-dynamic",
+  "version": "1.2.4",
+  "main": "./dist/index.cjs.js",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "private": true,
+  "publishConfig": {
+    "access": "public"
+  },
+  "backstage": {
+    "role": "backend-plugin",
+    "supported-versions": "1.27.7",
```

Comment:
Consider updating to `1.29.2`

Reply:
fair point as https://github.com/backstage/community-plugins/blob/main/workspaces/scaffolder-relation-processor/backstage.json is at 1.29.2

Reply:
```suggestion
    "supported-versions": "1.29.2",
```